src/lib/supabaseClient.ts

```diff
@@ -1,5 +1,4 @@
 import { createClient } from '@supabase/auth-helpers-sveltekit';
-// import type { Database } from './supabaseTypes';
```

Comment:
I'm not really sure how the `supabase` client knows that the `./supabaseTypes` file defines the TS types, but my type autocomplete didn't work when I delete that file. (that file was generated with `npx supabase gen types typescript --project-id $SUPABASE_PROJECT_ID --schema public > src/lib/supabaseTypes.ts`)